tests/rptest/services/redpanda.py

```diff
@@ -702,7 +702,8 @@ def start_redpanda(self, node):
             f"{preamble} {env_preamble} nohup {self.find_binary('redpanda')}"
             f" --redpanda-cfg {RedpandaService.NODE_CONFIG_FILE}"
             f" --default-log-level {self._log_level}"
-            f" --logger-log-level=exception=debug:archival=debug:io=debug:cloud_storage=debug "
+            f" --logger-log-level=exception=debug:archival=debug:io=debug:cloud_storage=debug:seastar_memory=debug "
+            f" --abort-on-seastar-bad-alloc "
```

Comment:
should we add something like `--dump-memory-diagnostics-on-alloc-failure-kind=all`? It looks like the default value is allocations marked as `critical`

https://github.com/scylladb/seastar/blob/master/src/core/reactor.cc#L3475

also nit: we can remove `f-strings` here as these are static strings.

Reply:
When `--abort...` is specified it will dump the diagnostics on any type of failure.

I will remove the f-strings.